### PR TITLE
Project deployed and README file updated

### PR DESCRIPTION
**Pull request by AbbasSarwar**
Hello, My Project is now live (https://abbassarwar.github.io/Abbas-Portfolio/)
### **Versions** 
**Mobile** 375px - 768px
**Desktop** 769px - 1920px

**Key features**
* Work Section
* Form section
* Colors 
* Li inputs for keyboard users
* Form validation

**BUGs**
* Sections fixed
* Form validation fixed
* screens fixed (fixation still needed)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +73,7 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 
-- [Live Demo Link]([https://github.com/AbbasSarwar/Mobile-First])(https://github.com/AbbasSarwar/Mobile-First)
+- [Live Demo Link]([https://abbassarwar.github.io/Abbas-Portfolio/)
```
Comment (OyePriscilla) [inline README.md]: - [x] This link is not '**clickable**', kindly format it properly as you did for the issues link in line 138. 👍 You may try the suggested code below. 
```suggestion
- [Live Demo Link](https://abbassarwar.github.io/Abbas-Portfolio/)
```

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @AbbasSarwar 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
App deployed properly ✔️ 
Use GitHub page ✔️ 
Website works as expected ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (AbbasSarwar) [reply README.md]: fixed.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @AbbasSarwar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 
-  App deployed correctly. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**